test/functional/p2p_invalid_messages.py

```diff
@@ -124,7 +124,8 @@ def run_test(self):
         #
         # Send messages with an incorrect data size in the header.
         #
-        actual_size = 100
+        # These all fail with a checksum error!  That's not what this test is supposed to test.
```

Comment:
Did you mean to commit all of this? It seems unfinished.

Reply:
Test changes moved to a separate PR